app/src/main/java/pl/edu/zut/mad/appwizut2/utils/SharedPrefUtils.java

```diff
@@ -0,0 +1,75 @@
+package pl.edu.zut.mad.appwizut2.utils;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+
+/**
+ * Klasa wspomagajaca prace z SharedPreferences
+ * 
+ * @author Sebastian Swierczek
+ * */
+public class SharedPrefUtils {
+
+    /**
+     * Metoda zwraca obiekt SharedPreferences, ktory wskazuje na plik
+     * SharedPreferences
+     * 
+     * 
+     * @param ctx
+     *            kontekst aplikacji
+     * 
+     * @return wskazany obiekt SharedPreferences
+     */
+    public static SharedPreferences getSharedPreferences(Context ctx) {
+	return ctx.getSharedPreferences(Constans.PREFERENCES_NAME,
+		Activity.MODE_PRIVATE);
```

Comment:
Hmm, PreferenceFragment normally writes to [PreferenceManager.getDefaultSharedPreferences()](http://developer.android.com/reference/android/preference/PreferenceManager.html#getDefaultSharedPreferences(android.content.Context)), there I'm storing selected tab setting too.